packages/grid/_modules_/grid/components/columnHeaders/ColumnHeaderFilterIcon.tsx

```diff
@@ -0,0 +1,65 @@
+import IconButton from '@material-ui/core/IconButton';
+import Badge from '@material-ui/core/Badge';
+import Tooltip from '@material-ui/core/Tooltip';
+import * as React from 'react';
+import { useGridSelector } from '../../hooks/features/core/useGridSelector';
+import { preferencePanelStateSelector } from '../../hooks/features/preferencesPanel/preferencePanelSelector';
+import { PreferencePanelsValue } from '../../hooks/features/preferencesPanel/preferencesPanelValue';
+import { useIcons } from '../../hooks/utils/useIcons';
+import { optionsSelector } from '../../hooks/utils/useOptionsProp';
+import { ApiContext } from '../api-context';
+
+export interface ColumnHeaderFilterIconProps {
+  counter?: number;
+}
+
+export const ColumnHeaderFilterIcon: React.FC<ColumnHeaderFilterIconProps> = (props) => {
```

Comment:
No `children` prop is supported

```suggestion
export const ColumnHeaderFilterIcon = (props: ColumnHeaderFilterIconProps) => {
```